Lps 169318 master test

evibudai opened the pull request.

evibudai committed:
LPS-169318 Calling getPermissionWherePredicate with null to trigger the old query with better performance

JhossephG committed:
LPS-198425 Add logic to support add translatable entries via UI in addObjectStringFieldEntryViaUI macro

JhossephG committed:
LPS-198425 Add addTranslatableTextEntryViaUI macro

JhossephG committed:
LPS-198425 Add BUTTON_FIELD_ENTRY_LANGUAGE and SELECT_FIELD_ENTRY_LANGUAGE paths

JhossephG committed:
LPS-198425 Add CanUseUniqueValue test

JhossephG committed:
LPS-198425 Add logic to support translatable entries in addObjectEntryViaAPI macro

achaparro committed:
LPS-197686 Fix an old issue

oliv-yu committed:
LPS-174758 LPS-180501 portal-search-web: Create new Site Activities options for searchbar config behind ff LPS-159643

oliv-yu committed:
LPS-174758 LPS-180501 portal-language-lang: New lang keys for Site Activities

oliv-yu committed:
LPS-174758 LPS-180501 portal-search-web: Import LearnMessage from frontend-js-components-web

oliv-yu committed:
LPS-174758 LPS-180501 portal-language-lang: gradlew buildLang

oliv-yu committed:
LPS-174758 LPS-180501 portal-impl: Add feature flag for LPS-176691

oliv-yu committed:
LPS-174758 LPS-180501 portal-search-web: Only four types are enabled for recent assets

oliv-yu committed:
LPS-174758 LPS-180501 portal-search-web: Only allow single selection for content type

oliv-yu committed:
LPS-174758 LPS-180501 portal-search-web: Update wording

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Turn contentType into a dropdown

Commit:
LPS-174758 LPS-200418 portal-search: Create Asah Individuals Configuration

Commit:
LPS-174758 LPS-200418 portal-search: Create new Base Class to individuals Suggestions

Commit:
LPS-174758 LPS-200418 portal-language-lang: Add language key

Commit:
LPS-174758 LPS-200418 portal-language-lang: BuildLang

Commit:
LPS-174758 LPS-200418 portal-search: Implement Contributors to Recent Searches, Recent Pages, Recent Sites and Recently Viewed entries to users leveraging individual activities data from Analytics Cloud

Commit:
LPS-174758 LPS-200418 portal-search: Implement AsahIndividualsWebCacheItem

Commit:
LPS-174758 LPS-200418 portal-search-api: Add Constants

Commit:
LPS-174758 LPS-200418 portal-search: Create new Base class so the code can be reused

Commit:
LPS-174758 LPS-200418 portal-search: Move Base classes to new package and remove duplicated code

Commit:
LPS-174758 LPS-200418 portal-search: Apply new way to use and move to new package

Commit:
LPS-174758 LPS-200418 portal-search: Create Base classe to webCacheItem

Commit:
LPS-174758 LPS-200418 portal-search: Use web cache base class

JhossephG committed:
LPS-198425 Update VIEW_ENTRY_FIELD_BY_INDEX to works correctly

JhossephG committed:
LPS-198425 Apply

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Fix missing value

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Create TimeRangeInput and add to contributor settings

oliv-yu committed:
LPS-174758 LPS-200692 portal-language-lang: Add helptext for time range

oliv-yu committed:
LPS-174758 LPS-200692 portal-language-lang: gradlew buildLang

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Fix value + relax ‘required’ status on time range

ces-quanhuynh committed:
LPS-189610 Configure the User Image size limit in virtual instance scope

Commit:
LPS-174758 LPS-200418 portal-search: Create SearchUtil to display assetURLView

ces-quanhuynh committed:
LPS-189610 Remove unused User Image size limit system configuration

ces-totrinh committed:
LPS-189610 baseline

achaparro committed:
LPS-197686 Register the error, add release, and repeat the upgrade when the schema version is 0.0.0

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Switch to Clay Picker and cleanup site activities options

javalosjr96 committed:
LPS-197686 change method name to getStatus and return status of the upgrade

achaparro committed:
LPS-197686 New modules with database changes  are major upgrades

achaparro committed:
LPS-197686 Manual SF

achaparro committed:
LPS-197686 Only ask for checking the logs when there is an upgrade and the status is somehow positive

achaparro committed:
LPS-197686 Let's try to return no upgrade when we really did not run any upgrade. We were missing the case when only an upgrade process was pending to run and it failed, the schema version remains the same so we returned no upgrade. However the result will return failure because there are pending upgrades to run so we can rely on that to return the type. Maybe it is not a major but, since the status is unstable, better be careful

evibudai committed:
Revert "TEST Lps 169318 master"

evibudai committed:
LPS-169318 test

LPS-169318 test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

evibudai commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11998/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11998/jenkins-report.html">here</a>.</b></p>

evibudai commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/5254/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5254/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/311037b12e341f0665f0a7af99c1d90484b61a90">311037b12e341f0665f0a7af99c1d90484b61a90</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11998/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11998/jenkins-report.html">here</a>.</h5><hr/><h4>Failures (upstream comparison is not available):</h4><ol><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11998/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/evibudai/liferay-portal/pull/8">evibudai/LPS-169318-master-2-test</a></strong><pre>Unable to rebase master-temp-1700298860447 to master
Rebasing (1/3)
warning: inexact rename detection was skipped due to too many files.
warning: you may want to set your merge.renamelimit variable to at least 16021 and retry the command.
error: could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn
Resolve all conflicts manually, mark them as resolved with
"git add/rm &lt;conflicted_files&gt;", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 2 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/311037b12e341f0665f0a7af99c1d90484b61a90">311037b12e341f0665f0a7af99c1d90484b61a90</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/5254/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5254/jenkins-report.html">here</a>.</h5><hr/><h4>Failures (upstream comparison is not available):</h4><ol><li><div><h5><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/5254/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/evibudai/liferay-portal/pull/8">evibudai/LPS-169318-master-2-test</a></strong><pre>Unable to rebase master-temp-1700299209311 to master
Rebasing (1/7)
warning: inexact rename detection was skipped due to too many files.
warning: you may want to set your merge.renamelimit variable to at least 16021 and retry the command.
error: could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn
Resolve all conflicts manually, mark them as resolved with
"git add/rm &lt;conflicted_files&gt;", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

javalosjr96 committed:
LPS-197686 apply changes

javalosjr96 committed:
LPS-197686 check for VerifyExceptions and add status for VerifyProcesses

javalosjr96 committed:
LPS-197686 apply updates to ReleaseManagerTest

albertojml committed:
LPS-202115 Creation and usage of new property so that Object exception mapper services can be initialized for other modules

Commit:
LPS-174758 LPS-200418: Remove unecessary blank validation

Commit:
LPS-197686 update existing tests

Commit:
LPS-197686 add testFailureResultByPendingModuleUpgrade

Commit:
LPS-197686 add testFailureResultByVerifyException

Commit:
LPS-197686 add testUnresolvedResultByUnsatisfiedComponent

amosfong committed:
ISSD-385 move to a shared module

Commit:
LPS-199641 Delete UpgradeJavaServiceTrackerListCheck

Commit:
LPS-199641 Update replacements.json

Commit:
LPS-199641 Add new test cases for UpgradeCatchAllCheck

Commit:
LPS-199641 Documentation

albertojml committed:
LPS-202115 Allow for the execution of POST endpoints without response schema

albertojml committed:
LPS-202115 Added new test for checking if the object exceptions are mapped correctly in the responses

albertojml committed:
LPS-202115 Added test for checking that POST endpoint without response schema work as expected

BarbaraCabrera committed:
LPS-202229 Create BookmarksConfigurationDisplayContext

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Fix SF failure

BarbaraCabrera committed:
LPS-202229 Add necessary methods to the configuration Display Context

BarbaraCabrera committed:
LPS-202229 Add new configuration jsp to show the new view and add this view under feature flag in BookmarksAdminConfigurationAction

BarbaraCabrera committed:
LPS-202229 Add the necessary elements for the new view and the imports

DiegoHu97 committed:
LPS-202763 Add aria-label to buttons in layout-reports-web

DiegoHu97 committed:
LPS-202763 Fix typo in segments-simulation-web

DiegoHu97 committed:
LPS-202763 Set activeLabel the first time that renders in segments-web

devpedrotavares committed:
LPS-203252 Make expressions work with null values

DiegoHu97 committed:
LPS-202763 Add forwardRef in segments-web

Commit:
LPS-174758 LPS-200418 portal-search: Rename searchUtil to AssetURLUtil

Commit:
LPS-174758 LPS-200418 portal-search: Refactor web cache item

Commit:
LPS-174758 LPS-200418 portal-search: Rename Contributors

LuisOrtiz89 committed:
LPS-197686 Better this way

LuisOrtiz89 committed:
LPS-197686 Some cleanup and removing type assertions in test cases fully focused on the result and where the type is confusing

Mikellorza committed:
LPS-202421 Add test to add an AddAssetAutoTaggerEntry with exiting asset tag

BarbaraCabrera committed:
LPS-202229 Add the EmailFrom validator under the FF

BarbaraCabrera committed:
LPS-202229 Move select folder code to propsTransformers to use Clay:button instead of aui:button

BarbaraCabrera committed:
LPS-202229 Move saveConfiguration code to propsTransformers to use Clay:buttons instead of aui:buttons

DiegoHu97 committed:
LPS-202763 Add aria-label to buttons in content-dashboard-web

petershin committed:
ISSD-385 Fix compile

petershin committed:
ISSD-385 Project dependency

DiegoHu97 committed:
LPS-202763 Update snapshots

amosfong committed:
ISSD-385 add transitive gradle build

amosfong committed:
ISSD-385 remove sf suppression

thektan committed:
LPS-191958 Rename modal shows error toast when response is not ok

thektan committed:
LPS-191958 Rename modal disables save when input is empty

thektan committed:
LPS-191958 Rename modal disables save when submitting

thektan committed:
LPS-191958 Refactor - rename for consistency

oliverkecskemety committed:
LPS-202716 Wrap all endpoints that does not handle ConflictException
We use ConflictException to show that an object was provisioned by another SCIM client, however this exception is only handled in Charon during create. We need to wrap the other endpoints to return the proper response instead of a runtime exception.

susanchen3 committed:
LPS-197686 Update upgrade status when core module has failed

susanchen3 committed:
LPS-197686 Remove Gogoshell connection handling because an irrelevant error is thrown

Mikellorza committed:
LPS-202421 Add test to add an AddAssetAutoTaggerEntry without exiting asset tag

Mikellorza committed:
LPS-202421 Move class to the correct package and use the correct class to use in the tests

albertojml committed:
LPS-202115 Added test for testing the behavior fixed byc4c0ad2862cca27017be4dda2db6bf7ac0390847

Mikellorza committed:
LPS-202421 Rename methods to have a better understanding of what are testing

Mikellorza committed:
LPS-202421 Improve code. Extract common code to a method

albertojml committed:
LPS-202115 Changed parameter name

Commit:
LPS-174758 LPS-200418 portal-search: Fix sort parameter

4lejandrito committed:
LPS-202115 SF

4lejandrito committed:
LPS-202115 Extracting logic

Commit:
LPS-174758 LPS-200418 portal-language-lang: Rename language key

Commit:
LPS-174758 LPS-200418 gw buildLang

Commit:
LPS-174758 LPS-200418 portal-search: minCounts default value must be 0

nicolas-sss committed:
LPS-199641 Improve UpgradeCatchAllCheck to handle cases with

4lejandrito committed:
LPS-202115 Simplifying even more

4lejandrito committed:
LPS-202115 SF

4lejandrito committed:
LPS-202115 SF

4lejandrito committed:
LPS-202115 Inlining

DiegoHu97 committed:
LPS-202763 Mock sub function to return string in content-dashboard-web

DiegoHu97 committed:
LPS-202763 Mock sub function to return string in asset-list-web

4lejandrito committed:
LPS-202115 No need to create a new object definition

4lejandrito committed:
LPS-202115 Not needed

susanchen3 committed:
LPS-197686 Update case to detect major upgrade status since entries for modules don't exist yet

susanchen3 committed:
LPS-197686 Update upgrade status since warning is no longer a valid status

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Set default minCounts to 0 for recent searches

mateussandes23 committed:
LPS-202410 Add virtual instance property on the tests

oliv-yu committed:
LPS-174758 LPS-200692 portal-search-web: Changing site activity type should reset inputs to default

nicolas-sss committed:
LPS-203017 Create the UpgradeJavaDDMFormValuesSerializerTrackerCheck

nicolas-sss committed:
LPS-203017 Create test for UpgradeJavaDDMFormValuesSerializerTrackerCheck

nicolas-sss committed:
LPS-203017 Generated

Commit:
LPS-199641 Fix in UpgradeCatchAllCheck file test

nicolas-sss committed:
LPS-200168 Follow up - SF - Removing unnecessary code

nicolas-sss committed:
LPS-198614 Fix on UpgradeCatchAll message generation

Commit:
LPS-203036 Always show the trash icon during ObjectEntries listing flow to improve performance

ccorreagg committed:
LPS-197569 Add new method to consider a zoneId instead of using one by default

ccorreagg committed:
LPS-197569 baseline

ccorreagg committed:
LPS-197569 Update usages to pass the user time zone

albertojml committed:
LPS-202115 SF

albertojml committed:
LPS-202115 Merge of test methods

4lejandrito committed:
LPS-202115 Much better

4lejandrito committed:
LPS-202115 Inlining

4lejandrito committed:
LPS-202115 Inlining

4lejandrito committed:
LPS-202115 Keeping naming consistency with get tests

Commit:
LPS-174758 LPS-200418 portal-search: Add site URL

nicolas-sss committed:
LPS-198614 SF - Remove unused code

carolmariaabb committed:
LPS-201461 Use XMLHttpRequest instead of axios because of known vulnerability

carolmariaabb committed:
LPS-201461 yarn.lock

Commit:
LPS-174758 LPS-200418 portal-search: Fix test

thektan committed:
LPS-191958 Directly use openDefaultFailureToast for code consistency

thektan committed:
LPS-191958 Only disable save button when saving

Commit:
LPS-155562 Use Search Index over Database to prevent full table scan due to treePath being LongText

4lejandrito committed:
LPS-202115 Inlining

4lejandrito committed:
LPS-202115 Not needed

4lejandrito committed:
LPS-202115 Creating a simpler object definition for this test

4lejandrito committed:
LPS-202115 Inlining

brianchandotcom committed:
LPS-174758 Wordsmith

lfbesada committed:
LPS-202932 Detects and adds the portlets embedded in the fragment's HTML using runtimeTag with freemarker

lfbesada committed:
LPS-202932 Adds the capability to support dynamic instanceIDs by replacing FragmentEntryLink namespace if present

lfbesada committed:
LPS-202932 Adds the required dependencies

lfbesada committed:
LPS-202932 Adds an integration tests to assert this use case

lfbesada committed:
LPS-202932 Adds an integration tests to assert the same use case but using a dynamic instance ID

lfbesada committed:
LPS-180573 Add an empty view template to the LayoutContentPageEditorWebTestPortlet

4lejandrito committed:
LPS-202115 Intellij added these ones

4lejandrito committed:
LPS-202115 SF

magjed4289 committed:
LPS-202532 Internal server error when a already existing ERC is used in a POST API call

Commit:
LPS-174758 LPS-200418 portal-search: Rename classes properly

Commit:
LPS-174758 LPS-200418 portal-language-lang: Add language Key

Commit:
LPS-174758 LPS-200418 buildLang

Commit:
LPS-174758 LPS-200418 portal-language-lang: Remove Language Key

Commit:
LPS-174758 LPS-200418 portal-search: Add logic to feature flag

Commit:
LPS-174758 LPS-200418 portal-search: Use new language Key

4lejandrito committed:
LPS-202532 External reference must be scoped by object definition

I am essentially restoring the original logic, just keeping the new
exception.

The new logic was not compatible as the uniqueness of the ERC was being
checked by group instead of by object definition.

With this change we keep the same logic just using the right exception.

Commit:
LPS-174758 LPS-200418 portal-search-web: Update Contributor names in FE

4lejandrito committed:
LPS-202532 Setting error message

BarbaraCabrera committed:
LPS-203027 Add onKeyDown for card and list items

mateussandes23 committed:
LPS-203036 Refactor PreventDeletionOneToManyWontAllowDelete and PreventDeletionManyToManyWontAllowDelete tests

mateussandes23 committed:
LPS-203036 Add assertDeleteEntryButtonIsPresent macro

interaminense committed:
LRAC-15102 Refactor experiment components, remove unused functions, components and unit tests and implement test traffic component

interaminense committed:
LRAC-15102 buildLang

DiegoHu97 committed:
LPS-202763 Remove required prop in segments-experiment-web

ealonso committed:
LPS-197569 Adds new test to validate that the content is created with the user time zone

DiegoHu97 committed:
LPS-202763 Add aria-label to button in asset-list-web

DiegoHu97 committed:
LPS-202763 Add aria-label to buttons in segments-experiment-web

marcellustavares committed:
LRAC-15095 Remove release flag

marcellustavares committed:
LRAC-15162 Track journal article resource primary key

marcellustavares committed:
LRAC-15165 Fix analytics-asset-type value for Document and Media

amosfong committed:
ISSD-385 rename

michaelhashimoto committed:
LRCI-3892 Use parameters to display in the Job Page

michaelhashimoto committed:
LRCI-3892 Use parameters on Create Job Page to avoid hard-coding

michaelhashimoto committed:
LRCI-3892 Convert the parameters into a string if available when creating a JobEntity

michaelhashimoto committed:
LRCI-3892 Add ParameterDefinitions for JobEntities in order to be passed to the UI

michaelhashimoto committed:
LRCI-3892 Define the getParameterDefinitions methods to be passed along to the UI

michaelhashimoto committed:
LRCI-3892 Rely on Job parameters instead of build parameters to determine the certain fields

michaelhashimoto committed:
LRCI-3892 SF

ling-alan-huang committed:
LPS-203330 Move package

ealonso committed:
LPS-191374 Revert previous changes

ealonso committed:
LPS-191374 From Tomáš: No site should be created as "OPEN" by default. The default value should be TYPE_SITE_RESTRICTED. This relates to the Guest site as well as new sites being created.

ealonso committed:
LPS-203059 Return null if the feature flag is enabled to avoid empty dropdown selector

victorg1991 committed:
LPS-203343 Add the form inside the body

lfbesada committed:
LPS-203344 Take into account that Robots is a free text box, as it could include spaces we should use an empty string as separator

Mikellorza committed:
LPS-202421 Add test to FindByG_C_N with case-sensitive tags

ealonso committed:
LPS-202932 Inline values

ealonso committed:
LPS-202932 Unused

ealonso committed:
LPS-202932 No needed

ealonso committed:
LPS-202932 Reorder

sandrodw3 committed:
LPS-202200 Add tests for conditions and actions description

Mikellorza committed:
LPS-202421 Add test to test CountByG_C_N with case-sensitive tags

Mikellorza committed:
LPS-202421 Extract code to _testFindByG_C_N to be reused

Mikellorza committed:
LPS-202421 Use injected component instead of util

lfbesada committed:
LPS-203344 Adds an integration test

DiegoHu97 committed:
LPS-202763 Mock navigation function in segments-experiment-web

DiegoHu97 committed:
LPS-202763 Fix typo in segments-experiment-web

DiegoHu97 committed:
LPS-202763 Mock function properly in content-dashboard-web

DiegoHu97 committed:
LPS-202763 buildLang

Commit:
COMMERCE-12579 update BREAKING_CHANGES_AMENDMENTS.markdown

AliciaGarciaGarcia committed:
LPS-199736 not needed

ealonso committed:
LPS-203148 Adds PermissionCheckerMethodTestRule

brianchandotcom committed:
LPS-189610 Rename

brianchandotcom committed:
LPS-189610 Remove unused overrides that do nothing

brianchandotcom committed:
ISSD-385 prep next

brianchandotcom committed:
LPS-202115 SF

brianchandotcom committed:
LPS-197686 SF

ealonso committed:
LPS-203148 Sets correct user to themeDisplay

ealonso committed:
LPS-203148 Fix null pointer exceptions

victorg1991 committed:
LPS-203386 Add aria-label to the selector

victorg1991 committed:
LPS-203386 Do not embed add button in the role menu

Commit:
LPS-198794 Delete testcase file unnecessary

Commit:
LPS-198794 Fix macro to add parameters after the clay changes

4lejandrito committed:
LPS-202532 Merging tests in 2 following the existing pattern

michaelhashimoto committed:
LRCI-3892 Rename constants

Commit:
LPS-174758 LPS-200418 portal-search: Rename variables

victorhcunha committed:
LPS-203407 Add run.type single for tests using Groovy script

Commit:
LPS-174758 LPS-200418 Refactor assetViewURL so the code can be reused arround components

Commit:
LPS-198794 Fix name from tasks of the CanCreateObjectWithRelationshipParameter and change the relationship

Commit:
LPS-198794 Fix name from tasks of the CanEditRelationshipParameter and change the relationship

ealonso committed:
LPS-203289 Return correct token

brianchandotcom committed:
LPS-174758 WIP

brianchandotcom committed:
LRCI-3892 SF

brianchandotcom committed:
LPS-202532 SF

brianchandotcom committed:
LPS-202532 Wordsmith

brianchandotcom committed:
LPS-203148 SF

Commit:
LPS-174758 LPS-200418 SF

brianchandotcom committed:
LPS-174758 More organized

brianchandotcom committed:
LPS-174758 SF

brianchandotcom committed:
LPS-174758 SF

brianchandotcom committed:
Baseline

ealonso committed:
LPS-198237 Order countries by localized name

Commit:
LPS-174758 LPS-200418 SF

brianchandotcom committed:
LPS-174758 SF

brianchandotcom committed:
LPS-202229 as used

Commit:
LRAC-14774 Change locator1 to value1

Commit:
LPS-203405 Modify cases to adapt new markup

brianchandotcom committed:
Revert "LPS-155562 Use Search Index over Database to prevent full table scan due to treePath being LongText"

This reverts commit ddce39223adafb4a9a70a941d2d22e912a6b4ece.

evibudai committed:
Merge branch 'LPS-169318-master-2-test' of github.com:evibudai/liferay-portal into LPS-169318-master-2-test

evibudai commented:
ci:test:relevant

evibudai commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/12234/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12234/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 2 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee2717f38bb6f171e594caaa0bcc70a934330ded">ee2717f38bb6f171e594caaa0bcc70a934330ded</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/03f7cfaae5c793b488d4684b421ba7c41770d1c0">03f7cfaae5c793b488d4684b421ba7c41770d1c0</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2471298666">EE Development Acceptance (master) - 223 - 2023-12-05[16:40:43]</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/12234/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12234/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/12234/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/evibudai/liferay-portal/pull/8">evibudai/LPS-169318-master-2-test</a></strong><pre>Unable to rebase master-temp-1701927713646 to master
Rebasing (1/10)
Rebasing (2/10)
Rebasing (3/10)
Rebasing (4/10)
warning: inexact rename detection was skipped due to too many files.
warning: you may want to set your merge.renamelimit variable to at least 16707 and retry the command.
error: could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn
Resolve all conflicts manually, mark them as resolved with
"git add/rm &lt;conflicted_files&gt;", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 7361a46c9ca86... LPS-169318 Overloading getPermissionWherePredicate with userIdColumn to be able to call the method with an explicitly given userIdColumn</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
